test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -951,7 +964,7 @@ UCS_TEST_P(test_uct_cm_sockaddr, conn_to_non_exist_ip)
     ucs_status_t status;
     size_t size;
 
-    skip_tcp_sockcm();
+    skip_tcp_sockcm();  /* error handling isn't implemented yet in tcp_sockcm  */
```

Comment:
can we use UCS_TEST_SKIP_COND_P + check iface err handling capability?

Reply:
not addressed?

Reply:
sorry, missed this one.
it's just a temp skip until error handling is implemented.
which flag do you want to check?

Reply:
IIUC this test is not relevant if `UCT_IFACE_FLAG_ERRHANDLE_PEER_FAILURE` is not supported. Such check  would be non-temp.

Reply:
@evgeny-leksikov I meant here that in tcp_sockcm the flow where there is some error and need to handle it, isn't implemented. if an error happens, then need to disconnect, for example, but disconnect isn't implemented yet. this is why it's temp

Reply:
i see, thanks!